core/block_svg.ts

```diff
@@ -140,6 +141,7 @@ export class BlockSvg extends Block implements IASTNodeLocationSvg,
   // Create core elements for the block.
   private svgGroup_: SVGGElement;
   style: BlockStyle;
+  /** @internal */
```

Comment:
Here you've added a one-liner (which is fine, but inconsistent with the example above).